File: Is it hot or cold.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int c;
        scanf("%d",&c);
        if(c>20)
        printf("HOT\n");
        else
        printf("COLD\n");
    }
}

File: Is it hot or cold.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int n=sc.nextInt();
            if(n<=20)
            System.out.println("COLD");
            else
            System.out.println("HOT");
        }
    }
}